test/runtests.jl

```diff
@@ -783,7 +783,6 @@ expr = JSON3.read("""
 @test JSON3.write(Int64) == "\"Int64\""
 @test JSON3.write(Float64) == "\"Float64\""
 @test JSON3.write(String) == "\"String\""
-@test startswith(JSON3.write(NamedTuple{(:a, :b), Tuple{Int64, String}}), "\"NamedTuple{(:a, :b),")
```

Comment:
why'd you delete this test?

Reply:
It fails on nightly because they changed how NamedTuple types stringify (they use the condensed macro form now); and this test wasn't really testing anything that unique; we have tests covering other similar stuff.